steward/src/cork/proposals.rs

```diff
@@ -0,0 +1,372 @@
+use std::time::Duration;
+
+use abscissa_core::{
+    tracing::{debug, error, info, log::warn},
+    Application,
+};
+use futures::executor::block_on;
+use gravity_bridge::gravity_proto::cosmos_sdk_proto::cosmos::base::{
+    query::v1beta1::PageRequest, tendermint::v1beta1::GetLatestBlockRequest,
+};
+use somm_proto::{
+    cork::ScheduledCorkProposal, cosmos_sdk_proto::cosmos::base::abci::v1beta1::TxResponse,
+};
+use steward_proto::steward::{governance_call::Call, GovernanceCall};
+use tokio::task::JoinHandle;
+
+use crate::{
+    cellars::{aave_v2_stablecoin, cellar_v1},
+    config::StewardConfig,
+    cork::schedule_cork,
+    error::Error,
+    prelude::APP,
+};
+
+type GovQueryClient = gravity_bridge::gravity_proto::cosmos_sdk_proto::cosmos::gov::v1beta1::query_client::QueryClient<tonic::transport::channel::Channel>;
```

Comment:
It looks like these two clients are normal Cosmos clients, but we're retrieving them in this indirect and roundabout way because we are already importing `gravity_bridge`. Should we directly import these ourselves? Is this something `deep_space` handles?

Reply:
iirc this is one of the side effects of the dependency spaghetti we have going on. importing it from another crate wasn't compiling because of version conflicts.